src/utilities/transition/BaseTransition.js

```diff
@@ -0,0 +1,250 @@
+'use strict';
+
+// Required modules.
+var EventObserver = require( '../../mixins/Events.js' );
```

Comment:
I actually already changed this locally. Hooray! It should be in my latest commit.